deluge/ui/console/commander.py

```diff
@@ -86,14 +93,24 @@ def print_help(f=None):
             parser.print_help()
             return
 
-        if not getattr(parser, "_exit", False):
-            try:
-                ret = self._commands[cmd].handle(options)
-            except Exception as ex:
-                self.write("{!error!} %s" % ex)
-                log.exception(ex)
-                import traceback
-                self.write("%s" % traceback.format_exc())
-                return defer.succeed(True)
-            else:
-                return ret
+        if getattr(parser, "_exit", False):
+            return
+        return options
+
+    def exec_command(self, options, *args):
```

Comment:
At first glance I'm not sure what the differences are between `do_command`, `parse_command` and `exec_command` are. I think updates to the docstrings might help.